sisl/viz/plotly/plots/pdos.py

```diff
@@ -287,7 +296,12 @@ def _read_TBtrans(self, root_fdf, tbt_nc):
         self.PDOS = tbt_sile.DOS(sum=False).data.T
         self.E = tbt_sile.E
 
```

Comment:
Here instead it would be a wiser choice to do something like this:
```python
geom = self.get_sile("root_fdf").read_geometry(output=True)
self.geometry = tbt_sile.read_geometry(atom=geom.atoms)
```
I think this should work. This is better since the cdf reading of the geometry does some nice things ;)

Reply:
Done!